gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ selenium = "4.8.1"
 testcontainers = "1.17.6"
 unboundid-ldapsdk = "6.0.8"
 bouncycastle = "1.70"
-javax-activation = "1.2.0"
+angus-activation = "2.0.0"
 kotlin = "1.8.10"
```

Comment:
`com.sun.activation:jakarta.activation` - is dead project. Current supported project is `org.eclipse.angus:angus-activation` (supported latest `jakarta.activation-api` version `2.1.1`)